packages/zowe-explorer/scripts/generatePoeditorJson.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Merges package.nls.json and bundle.l10n.json into a key-value JSON file that
+ * can be imported into POEditor.
+ */
+
+const fs = require("fs");
+const poeditorJson = {};
+const packageNls = require(__dirname + "/../package.nls.json");
+for (const [k, v] of Object.entries(packageNls)) {
+    poeditorJson[k] = { [v]: "" };
+}
```

Comment:
I assume we are doing this in order to keep our "categorization" (`keys` in package.nls.json).
And also, we must keep them in order to be able to correlate the translation back to its original location/key when running the `downloadPoeditorL10n.js`.

Does that sound right, or am I off base here? 😋

Reply:
That's exactly right 🙂 Another benefit of these "keys" is that POEditor uses them as a unique ID for the translation rather than the string itself.